canvas_modules/common-canvas/package.json

```diff
@@ -111,7 +119,7 @@
   },
   "jest": {
     "transformIgnorePatterns": [
-      "node_modules/(?!d3-*)"
+      "/node_modules/(?!(lezer-r|d3|d3-array|d3-axis|d3-brush|d3-chord|d3-color|d3-contour|d3-delaunay|d3-dispatch|d3-drag|d3-dsv|d3-ease|d3-fetch|d3-force|d3-format|d3-geo|d3-hierarchy|d3-interpolate|d3-path|d3-polygon|d3-quadtree|d3-random|d3-scale|d3-scale-chromatic|d3-selection|d3-shape|d3-time|d3-time-format|d3-timer|d3-transition|d3-zoom)/)"
```

Comment:
`d3-*` in following config didn't work. So I had to specify all d3 packages in the config.
```
transformIgnorePatterns": [
      "/node_modules/(?!(d3-*|lezer-r)/)"
],
```